CHANGELOG.md

```diff
@@ -13,8 +13,22 @@ next release
 
 ### UI Changes
 
+1.41.0 (2023-01-04)
```

Comment:
Given there are only bug fixes and minor improvements, I'm not sure if we should make this a minor release or a patch release. I went with minor release for now since that's the "promise" to users based on the release schedule. 

Happy to change this to a patch release if that should be the protocol to follow in this situation (and make a note in our release instructions to this effect).